quic/s2n-quic-core/src/datagram/default.rs

```diff
@@ -504,7 +504,7 @@ mod tests {
 
     #[test]
     fn record_capacity_stats() {
-        const SMOOTHED_PACKET_SPACE: f64 = 102.3193359375;
+        const SMOOTHED_PACKET_SPACE: usize = 102;
```

Comment:
nit: while you are here can you add a comment on what this test is doing (the math behind 102). its not obvious why we get 102